experimental/embeddings/scripts/evaluate_sequence_tagging.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import os
+import pickle
+
+import flair
+import torch
+from flair.embeddings import TransformerWordEmbeddings
+
+from experimental.embeddings import datasets
+from experimental.embeddings.tasks import sequence_tagging as st
+
+
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-d",
+        "--device",
+        help="Device for Flair e.g. cpu",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-e",
+        "--embedding",
+        help="Huggingface embedding model name",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-ds",
+        "--dataset",
+        help="Dataset name",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "--dataset_url",
+        help="Dataset download url",
+        type=str,
+        required=False,
+    )
+    parser.add_argument(
+        "-r",
+        "--root",
+        help="Root directory for output files",
+        type=str,
+        required=False,
+        default="resources/tagging/",
```

Comment:
path is not compatible with windows os

Reply:
All paths will be refactored for usage with Pathlib instead of os, so I will verify it afterwards.